projects/workflows-creator/package.json

```diff
@@ -62,40 +62,5 @@
     "registry": "https://registry.npmjs.org/",
     "access": "public",
     "directory": "dist"
-  },
-  "release": {
-    "branches": [
-      "master"
-    ],
-    "plugins": [
```

Comment:
where are we moving the release config to?

Reply:
this config was needed when we publish via semantic 
now to publish both packages simultaneously will use lerna for that.
So not needed